tests/002-send-receive.cpp

```diff
@@ -628,4 +628,64 @@ namespace oxen::quic::test
         }
     }
 
+    TEST_CASE("002 - BParser generic request handler", "[002][bparser][generic]")
+    {
+        Network test_net{};
+
+        auto [client_tls, server_tls] = defaults::tls_creds_from_ed_keys();
+
+        Address server_local{};
+        Address client_local{};
+
+        std::mutex mut;
```

Comment:
Unused and triggered a fail in CI [here](https://ci.oxen.rocks/oxen-io/oxen-libquic/503/5/3)